simulator-starter/src/main/java/org/citrusframework/simulator/http/SimulatorRestConfigurationProperties.java

```diff
@@ -73,6 +73,10 @@ public String getUrlMapping() {
         return urlMapping;
     }
 
+    public String[] getUrlMappings() {
+        return urlMapping == null ? new String [0] : urlMapping.split(",");
```

Comment:
imo `urlMapping` should already be a `List<String`. that way, Spring will infer the type for the `application.properties` file.